component/testrunner/testrunner.json

```diff
@@ -161,7 +161,7 @@
         },
         "uris" :
         {
-          "add-nocache-param" : true
+          "add-nocache-param" : false
```

Comment:
I'll think the caching change should be another PR then. It makes It difficult to approve. If this change is backed out I'll vote for this PR, bringing the promise PR forward.